packages/amplication-client/src/Entity/EntityListItem.tsx

```diff
@@ -95,7 +84,7 @@ export const EntityListItem = ({
       />
       <DataGridRow navigateUrl={`/${applicationId}/entities/${entity.id}`}>
         <DataTableCell className="min-width">
-          <LockStatusIcon enabled={Boolean(entity.lockedByUser)} />
+          {Boolean(entity.lockedByUser) && <LockStatusIcon enabled />}
```

Comment:
What does this change mean?

Reply:
The component have two states - enabled (orange) and "not enabled" (gray).
This behavior is still used in the app control toolbar when editing an entity. 

In the entity list, it was decided not to show the icons at all for entities without changes - so instead of showing the icon in the "disabled" state we are not showing it at all